integreat_cms/api/v3/pages.py

```diff
@@ -212,20 +228,26 @@ def children(request, region_slug, language_slug):
         # like in wordpress depth = 0 will return no results in this case
         depth = depth - 1
     result = []
+    # pylint: disable=unused-variable
+    public_region_pages, skipped_pages = request.region.pages.filter(
+        explicitly_archived=False
+    ).cache_tree(archived=False, language_slug=language_slug)
```

Comment:
Yes, indeed. It will mostly reduce bandwidth required for the time being. We then need a fast method that builds the tree for one root node / tree ID.

Reply:
This works. We now only create the tree for the mptt trees we actually need (AFAICT).